src/load.ts

```diff
@@ -108,6 +110,9 @@ export async function mergeDependency(
   cachePath: string = path.join(os.homedir(), '.fhir', 'packages'),
   log: LogFunction = () => {}
 ): Promise<FHIRDefinitions> {
+  if (version === 'latest') {
+    version = await exports.getLatestVersion(packageName, log);
```

Comment:
Is this prefixed with `exports.` for unit testing purposes (to allow it to be swapped w/ a mock)? If so, it might be worth adding a comment. I had to think about it a minute before I realized why it was there (if I'm even right).